### PR TITLE
Fixes for crash on reload

### PR DESCRIPTION
**Pull request by acoates-ms**
Follow up from #6289

These additional changes were found during partner testing of the fix against 0.63.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6336)

### CODE REVIEW

File: vnext/Microsoft.ReactNative/Modules/Animated/PropsAnimatedNode.h [hunk 1]
```diff
@@ -40,7 +40,7 @@ class PropsAnimatedNode final : public AnimatedNode {
 
   int64_t m_connectedViewTag{s_connectedViewTagUnset};
   std::unordered_map<int64_t, comp::CompositionAnimation> m_expressionAnimations{};
-  std::vector<int64_t> m_suspendedExpressionAnimationTags{};
+  // std::vector<int64_t> m_suspendedExpressionAnimationTags{};
```
Comment (rectified95) [inline vnext/Microsoft.ReactNative/Modules/Animated/PropsAnimatedNode.h:43]: Are you planning another PR to adjust this or should the comment go away in this one?

**Review by rectified95 — APPROVED** [review]

File: vnext/Microsoft.ReactNative/Modules/Animated/PropsAnimatedNode.h (hunk 1 above)
Comment (acoates-ms) [reply vnext/Microsoft.ReactNative/Modules/Animated/PropsAnimatedNode.h:43]: This code was already commented out... I just commented out the rest of it.  This is tracked by issue #3280.  I assume the plan is to try to re-enable this code at some point.

File: vnext/Microsoft.ReactNative/Modules/Animated/PropsAnimatedNode.cpp [hunk 2]
```diff
@@ -164,22 +165,25 @@ void PropsAnimatedNode::DisposeCompletedAnimation(int64_t valueTag) {
...
     }
   }
+  */
 }
 
 void PropsAnimatedNode::ResumeSuspendedAnimations(int64_t valueTag) {
+  /*
   const auto iterator =
       std::find(m_suspendedExpressionAnimationTags.begin(), m_suspendedExpressionAnimationTags.end(), valueTag);
   if (iterator != m_suspendedExpressionAnimationTags.end()) {
```
Comment (NikoAri) [inline vnext/Microsoft.ReactNative/Modules/Animated/PropsAnimatedNode.cpp:179]: (nit) should all this code be removed?

**Review by NikoAri — APPROVED** [review]
:shipit:

**Review by vmoroz — APPROVED** [review]
:shipit:

**Pull request merged**